package.json

```diff
@@ -21,11 +21,12 @@
     "algoliasearch": "^4.14.2",
     "dotenv": "^16.0.3",
     "framer-motion": "^6.5.1",
+    "glob": "^9.3.4",
     "glob-promise": "^6.0.0",
     "gray-matter": "^4.0.3",
     "lucide-react": "^0.92.0",
     "next": "^13.1.5",
-    "next-mdx-remote": "^4.2.0",
+    "next-mdx-remote": "^4.4.1",
```

Comment:
Were you able to get it working with 4.4.1? I kept having an issue, I reverted to 4.2.0 after reading others having the same issue.

Reply:
Woops, that was not intended to be pushed. Was experimenting with the infinite load I was having. Reverted this.